app.rb

```diff
@@ -0,0 +1,68 @@
+require 'sinatra'
+require 'ood_core'
+require 'moab'
+
+Dir[File.dirname(__FILE__) + "/lib/*.rb"].each {|file| require_relative file }
+
+# more details see ood_appkit lib/ood_appkit/configuration.rb
+begin
+  CLUSTERS = OodCore::Clusters.new(OodCore::Clusters.load_file(ENV['OOD_CLUSTERS'] || '/etc/ood/config/clusters.d').select(&:job_allow?)
+            .select { |c| c.custom_config[:moab] }
+            .select { |c| c.custom_config[:ganglia] }
+            .reject { |c| c.metadata.hidden }
+          )
+rescue OodCore::ConfigurationNotFound
+  CLUSTERS = OodCore::Clusters.new([])
+end
+
+helpers do
+  def h(text)
+    Rack::Utils.escape_html(text)
+  end
+
+  def dashboard_title
+    ENV['OOD_DASHBOARD_TITLE'] || "Open OnDemand"
+  end
+
+  def dashboard_url
+    "/pun/sys/dashboard/"
+  end
+
+  def relative_url

```

Comment:
This method should not be needed; use Sinatra's `url(` helper for app url's in the views. For example, url('/') should take you to the root of the app.